README.rst

```diff
@@ -279,6 +279,25 @@ Install Python bindings
 
 .. installation-end-marker-do-not-remove
 
+
+Testing Dali build
+^^^^^^^^^^^^^^^^^^
+For the purpose of testing Dali, we maintain DALI_extra_ repository, where all testing data can be found.
+Should you want to run tests for Dali build, please obtain DALI_extra_ and put path to root repo folder
```

Comment:
Testing data can be found in the `DALI_extra` repository. This repo is maintained for testing DALI. To run these tests, first set the environment variable `DALI_EXTRA_PATH` to point to the root repo folder of `DALI_extra`.  The DALI testing module reads this environment variable during runtime.